src/iexamine.cpp

```diff
@@ -4904,6 +4907,27 @@ void iexamine::autodoc( player &p, const tripoint &examp )
             break;
         }
 
+        case RAD_AWAY: {
+            patient.moves -= 500;
+            patient.add_msg_player_or_npc( m_info,
+                                           _( "The autodoc scanned you and detected a radiation level of %d mSv (%d mSv from items)." ),
+                                           _( "The autodoc scanned <npcname> and detected a radiation level of %d mSv (%d mSv from items)." ),
+                                           patient.get_rad(), patient.leak_level( "RADIOACTIVE" ) );
```

Comment:
Perhaps (for immersion reasons) autodoc could suggest undergoing decontamination process `if ( patient.leak_level( "RADIOACTIVE" ) > 0 )` aka getting rid of radioactive item.

Reply:
Good idea, thanks.